src/config.ts

```diff
@@ -1,12 +1,4 @@
 export const networkConfigs = {
-  mainnet: {
-    label: 'Agoric Mainnet',
-    url: 'https://main.agoric.net/network-config',
-  },
-  testnet: {
-    label: 'Agoric Testnet',
-    url: 'https://testnet.agoric.net/network-config',
-  },
```

Comment:
out of scope for this PR

Reply:
Reverted and moved to https://github.com/Agoric/dapp-inter/pull/67